src/template-cache.js

```diff
@@ -74,16 +74,15 @@ _.extend(Marionette.TemplateCache.prototype, {
   // using a template-loader plugin as described here:
   // https://github.com/marionettejs/backbone.marionette/wiki/Using-marionette-with-requirejs
   loadTemplate: function(templateId, options) {
-    var template = Backbone.$(templateId).html();
+    var $template = Backbone.$(templateId);
```

Comment:
i know this is not directly related to this PR... but can anyone think of a reason we create a jquery object here only to call html on it?